plugins/woocommerce/src/Blocks/BlockTypes/ProductFilterStockStatus.php

```diff
@@ -260,4 +214,42 @@ function ( $stock_count ) {
 			}
 		);
 	}
+
+	/**
+	 * Register pattern for default product attribute.
+	 */
+	public function register_block_patterns() {
+		register_block_pattern(
+			'woocommerce/default-stock-filter',
+			array(
+				'title'    => '',
+				'inserter' => false,
+				'content'  => '
+<!-- wp:woocommerce/product-filter-stock-status -->
+<div class="wp-block-woocommerce-product-filter-stock-status">
+  <!-- wp:group {"metadata":{"name":"Header"},"style":{"spacing":{"blockGap":"0"}},"layout":{"type":"flex","flexWrap":"nowrap"}} -->
+  <div class="wp-block-group">
+    <!-- wp:heading {"level":3} -->
+    <h3 class="wp-block-heading">Status</h3>
+    <!-- /wp:heading -->
+    <!-- wp:woocommerce/product-filter-clear-button {"lock":{"remove":true,"move":false}} -->
+    <!-- wp:buttons {"layout":{"type":"flex"}} -->
+    <div class="wp-block-buttons">
+      <!-- wp:button {"className":"wc-block-product-filter-clear-button is-style-outline","style":{"border":{"width":"0px","style":"none"},"typography":{"textDecoration":"underline"},"outline":"none","fontSize":"medium"}} -->
+      <div class="wp-block-button wc-block-product-filter-clear-button is-style-outline" style="text-decoration:underline"><a class="wp-block-button__link wp-element-button" style="border-style:none;border-width:0px">Clear</a></div>
+      <!-- /wp:button -->
+    </div>
+    <!-- /wp:buttons -->
+    <!-- /wp:woocommerce/product-filter-clear-button -->
+  </div>
+  <!-- /wp:group -->
+  <!-- wp:woocommerce/product-filter-chips {"lock":{"remove":true}} -->
+  <div class="wp-block-woocommerce-product-filter-chips wc-block-product-filter-chips"></div>
+  <!-- /wp:woocommerce/product-filter-chips -->
+</div>
+<!-- /wp:woocommerce/product-filter-stock-status -->
+				',
+			)
+		);
+	}
```

Comment:
We need a pattern for the attribute filter because the product attribute is site data and it is linked to a block attribute. But I don't think we need one for the Stock filter. What do you think?